README.md

```diff
@@ -191,6 +191,14 @@ http
 
 </details>
 
+## UI tests
+
+Run UI tests with Playwright in parallel in Chrome (headed mode).
+
+```bash
+$ npm run test:e2e
```

Comment:
I want it to become part of the cli and not standalone
Something like:
`npm test -- -s e2e`

Reply:
yes for now we just need to have it working with a starting test.
Then we need to understand how reliable those are, but definitely will end up in the CI.